fixtures/ikan/stryder-sfb150.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Stryder SFB150",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2024-05-02",
+    "lastModifyDate": "2024-05-02"
+  },
+  "links": {
+    "manual": [
+      "https://ikancorp.com/Downloads/SFB150-QuickStartGuide.pdf"
+    ],
+    "productPage": [
+      "https://ikancorp.com/shop/studio-broadcast/dmx-lights-studio-broadcast/stryder-fanless-bi-color-2800k-6500k-150-watt-led-fresnel-light-w-dmx/"
+    ]
+  },
+  "physical": {
+    "dimensions": [482, 254, 254],
+    "weight": 5.9,
```

Those are shipping weight/dimensions.  Fixture dimensions (not including control box?) are a bit less.

```suggestion
    "dimensions": [406, 254, 178],
    "weight": 4.8,
```